client/src/components/EmotionForm.jsx

```diff
@@ -33,7 +31,7 @@ const EmotionForm = ({ audioFeatures, savedTracks }) => {
     "enthusiasm",
     "ecstasy",
   ];
-  //initialize state to manage user selection and formData
+  //initialize states to manage user selection and formData
   const [selectedEmotion, setSelectedEmotion] = useState("");
   const [valenceScore, setValenceScore] = useState("");
   const [decision, setDecision] = useState("");
```

Comment:
Decisions can only be two things right now: 
`Embrace` or `Let it go`. If you decide to use typescript in the future, you can create a "type" for decision for this which will limit another string being set as the decision
Check out typescript: https://www.typescriptlang.org/